docs/make.jl

```diff
@@ -1,4 +1,4 @@
-using Manifolds, ManifoldsBase, Documenter
+using Manifolds, ManifoldsBase, Documenter, JSON
```

Comment:
Likewise, do we still need this import?
```suggestion
using Manifolds, ManifoldsBase, Documenter
```

Reply:
Thanks for noticing, no we don't.